src/Form/CaseFilterType.php

```diff
@@ -213,6 +213,11 @@ function (FormEvent $event) use ($formModifier) {
                         switch ($filterChoice) {
                             case CaseSpecialFilterStatuses::IN_HEARING:
                                 // TODO: When hearing implemented: modify query builder correctly with cases having an active hearing
+                                $qb = $filterQuery->getQueryBuilder();
```

Comment:
Can the “TODO” line above this be removed now?

Reply:
Removed in c22aeb68fb47a45b67c0be2f7a44b145efa208dc.